### PR TITLE
EPD-2997 |  API Docs v2 | Create Homepage UI

### PR DESCRIPTION
**Pull request by rob-dumont**
Changes:
- Created UI for the home page 

Notes:
- Did not include the plane-circle images, those will be added in later when we have more time. Same with the dividers between the four metric items.

### CODE REVIEW

**Comment by mschembri-TN** [comment]
Task linked: [EPD-2997 API Docs | Create Homepage UI](https://app.clickup.com/t/14197839/EPD-2997)

File: trip-ninja-docs/src/common/ConnectWithSales.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <div className="col-8">
+        <h3>Not set up with Trip Ninja?</h3>
+        <p className="body-text mb-0">
+          Don’t miss out on the unbeatable results from our API, talk to our sales 
+          team to kickstart your Trip Ninja experience.
+        </p>
+      </div>
+      <div className="col-4 my-auto text-end">
+        <a 
+          href="mailto:support@tripninja.io?subject=Connect with Sales - Developer Questions"
```
Comment (simheather) [inline trip-ninja-docs/src/common/ConnectWithSales.tsx:17]: more of a question than a comment but what's the desired behaviour of opening a page when clicking on this link? In my Navbar branch i have it opening to a new tab by setting `target="_blank"` in the <a> tag. Not sure which is best, new tab or same tab?

Comment (rob-dumont) [reply trip-ninja-docs/src/common/ConnectWithSales.tsx:17]: Yeah I think opening in new tab is best!

File: trip-ninja-docs/src/index.css [hunk 2]
```diff
@@ -36,13 +37,18 @@ h3 {
   font-weight: 600;
 }
 
+a:hover {
```
Comment (simheather) [inline trip-ninja-docs/src/index.css:43]: lolo i was setting this by the css class, this makes more sense! will refactor mine when this is merged 🙌

**Review by simheather — APPROVED** [review]

**Review by abdallahkaram1994 — APPROVED** [review]

**Comment by vickchungg** [comment]
LocalQA approved

**Review by vickchungg — APPROVED** [review]

**Pull request merged**